Content.YAMLLinter/Program.cs

```diff
@@ -90,13 +90,22 @@ private int Run()
             {
                 var newErrors = val.Where(n => n.AlwaysRelevant).ToHashSet();
                 if (clientErrors.TryGetValue(key, out var clientVal))
-                {
                     newErrors.UnionWith(val.Intersect(clientVal));
-                    newErrors.UnionWith(clientVal.Where(n => n.AlwaysRelevant));
-                }
```

Comment:
I don't fully understand the reasoning behind `AlwaysRelevant` errors, and there doesn't seem to be any documentation.
Is the idea meant to be:
- Include all always-relevant server & client errors
- Only include sometimes-relevant errors if they exist both for the client and server?

Because currently it wasn't doing that. The old behaviour was that it includes not-always-relevant client errors, only if the server had a corresponding always relevant error? If that is intentional, should I also update it so that always-relevant client errors include not-always-relevant server errors?